### ISSUE TITLE
Error: Invalid taxonomic rank.

### ISSUE THREAD

**Issue by RicardoPluto** [bbuchfink/diamond#366]
Hello,
I used the downloaded nodes.dmp (from ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdmp.zip) when building the diamond database. However, I got an error: 
Loading taxonomy nodes...  [0.026s]
Error: Invalid taxonomic rank.
How can I solve this error?
Thank you!
[nodes.tar.gz](https://github.com/bbuchfink/diamond/files/4823244/nodes.tar.gz)

**Comment by bbuchfink** [comment]
There have been some new ranks added by NCBI. Please try to use the latest version of Diamond.

**Comment by RicardoPluto** [comment]
The problem has been solved, thank you very much！

**Comment by RicardoPluto** [comment]
Hi,
I used taxonmap, taxonnodes and taxonnames to build the database, but after blastp the result file (BLAST tabular format) does not contain species information. I checked the information in the command line options (http://www.diamondsearch.org/index.php?pages/command_line_options/) and found that BLAST tabular format (default) could get the information of staxids, sscinames, sskingdoms, kingdoms, sphylums, but it did not indicate what parameters can be added to get these information of staxids, sscinames, sskingdoms, kingdoms, sphylums.
So I want to know what additional parameters can get this information in BLAST tabular format?
Thank you!
